FirebaseAdmin/FirebaseAdmin.Tests/Auth/FirebaseUserManagerTest.cs

```diff
@@ -71,6 +71,63 @@ public void TooLargeClaimsPayload()
             Assert.Throws<ArgumentException>(() => new UserRecord("user1") { CustomClaims = customClaims });
         }
 
+        [Fact]
+        public async Task GetUserById()
+        {
+            var handler = new MockMessageHandler()
+            {
+                Response = new UserRecord("user1"),
+            };
+            var factory = new MockHttpClientFactory(handler);
+            var userManager = new FirebaseUserManager(
+                new FirebaseUserManagerArgs
+                {
+                    Credential = MockCredential,
+                    ProjectId = MockProjectId,
+                    ClientFactory = factory,
+                });
+            UserRecord userRecord = await userManager.GetUserById("user1");
+            Assert.Equal("user1", userRecord.Uid);
+        }
+
+        [Fact]
+        public async Task GetUserByIdUserNotFound()
+        {
+            var handler = new MockMessageHandler()
```

Comment:
This service actually responds with an empty array when the user is not found. We'll have to update the test case to reflect that at some point.

Reply:
Gotcha 👍